lib/screens/questionaire_screen/widgets/doctor_card.dart

```diff
@@ -46,20 +46,23 @@ class DoctorCard extends StatelessWidget {
     // Class init.
     if (_label == null) {
       switch (Platform.localeName) {
-        case 'de_DE':
+        case 'en_US':
+        case 'en-US':
+        case 'en_GB':
+        case 'en-GB':
```

Comment:
I don't know if it's overkill to add these locale codes:

'en_AU' or 'en-AU' - English (Australia)
'en_CA' or 'en-CA' - English (Canada)
'en_NZ' or 'en-NZ' - English (New Zealand)
'en_IN' or 'en-IN' - English (India)
'en_IE' or 'en-IE' - English (Ireland)
'en_ZA' or 'en-ZA' - English (South Africa)

Lol

But I tested your PR and works quite well :)